app/views/keywords/search.html.erb

```diff
@@ -0,0 +1,23 @@
+<div>
+  <div class="mb-2">
+    <p data-testid="search_result_search_term"><%= I18n.t('keywords.search_term') %>: <%= search_params[:search] %></p>
+    <p data-testid="search_result_query_type"><%= I18n.t('keywords.query_type') %>: <%= search_params[:query_type] %></p>
+  </div>
+
+  <p data-testid="search_result_total_matched_urls">
+    <%= I18n.t('keywords.total_matched_urls') %>: <%= search_result_presenter.total_urls %>
```

Comment:
@Goose97 just FYI, we can avoid the `:` by using the I18n feature like

```
I18n.t('keywords.total_matched_urls', total_urls: search_result_presenter.total_urls)
```

In the YML file, we will define it

```
total_matched_urls: 'Total matched URLs %{total_urls}'
```

Reply:
TIL 🙇 . Fixed in 91a4eef8138475aa7dfa586f30fbd1f45b1dbc04